Marlin/pins_PRINTRBOARD_REVF.h

```diff
@@ -117,17 +154,54 @@
 // Temperature Sensors
 //
 #define TEMP_0_PIN          1   // Analog Input (Extruder)
+#define TEMP_A_PIN          2   // Analog Input (Extrudrboard A THERM)
+#define TEMP_B_PIN          3   // Analog Input (Extrudrboard B THERM)
 #define TEMP_BED_PIN        0   // Analog Input (Bed)
 
 //
 // Heaters / Fans
 //
 #define HEATER_0_PIN       15   // C5 PWM3B - Extruder
-#define HEATER_1_PIN       44   // F6
-#define HEATER_2_PIN       45   // F7
+// Note that Printrboard REV F6 removed HEATER_A_PIN from EXP1 and
+// reassigned it for a second fan output.
+#define HEATER_A_PIN       44   // F6 - Extrudrboard A HOTEND
+#define HEATER_B_PIN       45   // F7 - Extrudrboard B HOTEND
 #define HEATER_BED_PIN     14   // C4 PWM3C
 
 #define FAN_PIN            16   // C6 PWM3A
+#define PRINTRBOARD_F6_HEATER_FAN_PIN 44 // pin F6 on REV F6, see above.
+
+#ifndef NO_EXTRUDRBOARD
```

Comment:
You may want to double check these pin assignments. 
Has anyone tested these pin assignments on a Printrboard Rev F with an Extrudrboard connected to Exp1?  I have this configuration set up and the my pin assignments do not match the above configuration.

Maybe this configuration should be a define for Scott's extrudrboard2